benchmarks/cuda/toy_detector_cuda.cpp

```diff
@@ -84,8 +82,10 @@ BENCHMARK_F(ToyDetectorBenchmark, CUDA)(benchmark::State& state) {
     traccc::cuda::fitting_algorithm<device_fitter_type> device_fitting(
         fitting_cfg, mr, async_copy, stream);
 
+    // Copy detector to device
+    auto det_fixed_buff = detray::get_buffer(det, device_mr, copy);
```

Comment:
Not sure why it's `det_fixed_buff`. I kept looking for what you "fixed" in it. 🤔 Just using `det_buff` would be a bit better of a name I think.

Reply:
I think I just copied that from somewhere where it meant "fixed size buffer". I will rename it